articlequality/utilities/extract_scores.py

```diff
@@ -83,6 +87,12 @@ def main(argv=None):
         level=logging.DEBUG if args['--debug'] else logging.INFO,
         format='%(asctime)s %(levelname)s:%(name)s -- %(message)s'
     )
+    if args['--class-weight'] is not None:
+        class_weights = dict(
+            map(_parse_class_weight_option, args['--class-weight'])
+        )
+        global CLASS_WEIGHTS
+        CLASS_WEIGHTS = class_weights
```

Comment:
This would completely override it, maybe you can use var.update to retain most of them?

Reply:
That's the idea. What is to be gained from retaining some of them when one is already providing some weights?

I've updated the commit as requested.